src/main/scala/scalaz/nio/ClientServerDemo.scala

```diff
@@ -0,0 +1,69 @@
+package scalaz.nio
+
+import java.io.IOException
+import java.net.InetSocketAddress
+
+import scalaz.nio.channels.{ AsynchronousServerSocketChannel, AsynchronousSocketChannel }
+import scalaz.zio.console._
+import scalaz.zio.{ App, IO }
+
+import scala.concurrent.duration._
+
+object ClientServerDemo extends App {
```

Comment:
This looks great, i think we can use it for _compiled_ documentation, e.g. tut

Reply:
can you pls move it inside here https://github.com/scalaz/scalaz-nio/blob/master/src/main/tut/channel.md